src/Backend/Modules/MediaLibrary/Domain/MediaFolder/Exception/MediaFolderNotFound.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Backend\Modules\MediaLibrary\Domain\MediaFolder\Exception;
+
+class MediaFolderNotFound extends \Exception
+{
+    public static function forEmptyId()
+    {
+        return new \Exception('The id you have given is null');
```

Comment:
why not 
```php
return new self('The id you have given is null');
```

I see you use a lot of times the generic `\Exception` class, while custom exceptions are way better because they are more explicit